packages/icons/src/stories/classic-leg-icon.story.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
Is there not a way to output the leg- and mode-icon stories as they previously were handled? This new code just seems like a lot of extra files/boilerplate.

Reply:
There is not a way to output as they were before since there is no way to dynamically generate a lot of stories once `storiesOf` is deprecated. See https://github.com/storybookjs/storybook/issues/9828. The previous way was also quite difficult to browse and somewhat meaningless. This new method shows the output of the various LegIcon and ModeIcon components which I feel is an improvement.